transactions/src/main/resources/schema/wildfly-txn_3_1.xsd

```diff
@@ -0,0 +1,337 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's not a problem for me but is there a precedent for this approach and if not can it be defined somewhere?